ajax/libs/swiftclick/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "swiftclick",
+  "description": "Eliminates the 300ms click event delay on touch devices that support orientation change",
+  "author": {
+    "name": "Ivan Hayes",
+    "url": "http://www.ivanhayes.com"
+  },
+  "homepage": "https://github.com/munkychop/swiftclick",
+  "keywords": [
+    "swiftclick",
+    "fastclick",
+    "mobile",
+    "tap",
+    "touch",
+    "click",
+    "delay"
+  ],
+  "filename": "swiftclick.min.js",
+  "version": "1.4.1",
+  "autoupdate": {
```

Comment:
As stated in Pull Request [#6](https://github.com/munkychop/swiftclick/pull/6), the author has refactored [swiftclick](https://github.com/munkychop/swiftclick) so that now both, the non-minified version (`swiftclick.js`) and the minified version (`swiftclick.min.js`), are published on the npm package ([swiftclick](https://www.npmjs.com/package/swiftclick)). The new version of the npm package has been published on npm.

git auto update has been replaced with npm auto update in commit 7c05cd7.